stdlib/SuiteSparse/src/umfpack.jl

```diff
@@ -176,6 +176,65 @@ lu(A::Union{SparseMatrixCSC{T},SparseMatrixCSC{Complex{T}}};
     "dense LU.")))
 lu(A::SparseMatrixCSC; check::Bool = true) = lu(float(A); check = check)
 
+"""
+    lu!(F::UmfpackLU, A::SparseMatrixCSC; check = true) -> F::UmfpackLU
```

Comment:
I know this not done in many other places in this file, but I thought that some Julia style guide asks not to have spaces when setting keyword arguments, also in line 216.